spyder/widgets/github/backend.py

```diff
@@ -159,9 +164,44 @@ def send_report(self, title, body, application_log=None):
                         self.gh_owner, self.gh_repo, issue_nbr))
             return True
 
-    def get_user_credentials(self): 
+    def _get_credentials_from_settings(self):
+        """Get the stored credentials if any."""
+        remember = CONF.get('main', 'report_error/remember_me')
+        username = CONF.get('main', 'report_error/username')
+        if not remember:
+            username = ''
+
+        return username, remember
+
+    def _store_credentials(self, username, password, remember=False):
+        """Store credentials for future use."""
+        if username and password:
+            CONF.set('main', 'report_error/username', username)
+            try:
+                keyring.set_password('github', username, password)
+            except RuntimeError:  # pragma: no cover
+                _logger().warn(_('failed to save password in keyring, you '
```

Comment:
Please change this to use a `QMessageBox.warning`, with this message:

> It was not possible to securely save your password. You will be prompted for
your Github credentials next time you want to report an issue.